### PR TITLE
Optimized EIS Module Queries

### PR DESCRIPTION
**Pull request by ankits2206-github**

### CODE REVIEW

File: FusionIIIT/applications/eis/views.py [hunk 1]
```diff
@@ -967,14 +965,14 @@ def editconference(request):
         uploaded_file_url = fs.url(filename)
         eis.paper=uploaded_file_url
     except:
-        print("nothing")
+        pass
```
Comment (dynamoh) [inline FusionIIIT/applications/eis/views.py]: Throw proper errors - Use loggers

**Review by dynamoh — CHANGES_REQUESTED** [review]

File: FusionIIIT/applications/eis/views.py (hunk 1 above)
Comment (ankits2206-github) [reply FusionIIIT/applications/eis/views.py]: Done .

**Review by adi0311 — CHANGES_REQUESTED** [review]
Remove these print statements:-
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L421
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L427
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L462
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L482
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L521
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L534
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L568
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L651
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L677
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L742
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L801
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L816
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L844
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L871
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1029
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1047
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1048
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1059
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1130
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L1135

**Review by adi0311 — CHANGES_REQUESTED** [review]
Optimize the import by merging these lines:-
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L17
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L19
https://github.com/FusionIIIT/Fusion/blob/960ef69f778aab5c1fd2f33fee7687cfc14862a5/FusionIIIT/applications/eis/views.py#L20

**Review by dynamoh — APPROVED** [review]

**Pull request merged**